features/migrate_client_to_minion.feature

```diff
@@ -0,0 +1,30 @@
+# Copyright (c) 2017 SUSE LLC
+# Licensed under the terms of the MIT license.
+
+Feature: Migrate a traditional client into a salt minion
+
+  Scenario: Migrate a sles client into a salt minion
+     Given I am authorized
+     When I follow "Salt"
+     Then I should see a "Bootstrapping" text
+     And I follow "Bootstrapping"
+     Then I should see a "Bootstrap Minions" text
+     # sle-client = traditional sles client
+     And I enter the hostname of "sle-client" as hostname
+     And I enter "22" as "port"
+     And I enter "root" as "user"
+     And I enter "linux" as "password"
+     And I select "1-SUSE-PKG-x86_64" from "activationKeys"
+     And I click on "Bootstrap"
+     And I wait for "100" seconds
+     Then I should see a "Successfully bootstrapped host! Your system should appear in System Overview shortly." text
+
+  Scenario: Verify the minion was bootstrapped with activation key
+     Given I am on the Systems overview page of this "sle-client"
+     Then I should see a "Activation Key: 	1-SUSE-PKG" text
+
+  Scenario: Check that service nhsd has been stopped
+     # bsc#1020902 - moving from traditional to salt with bootstrap is not disabling rhnsd
+     When I run "systemctl status nhsd | grep Active" on "sle-client"
```

Comment:
As for reusing the same VM for centos7 (bug 1012956), it's a nice idea. However, this creates a problem: you will need to migrate it FIRST, then do the normal tests with the resulting minion. With the SLE12, we currently do all tests, then migrate it AFTER. We can have two different cucumber features at two different points of time, but I don't like the idea, I'd prefer to group all tests of migration in one single feature. I have no good solution at this point.

As for renaming sle-client, if you don't mind I'll do in next days a big cleanup of these:
- good English sle-client => SLE client
- group all switch / if... elsif statements in one ruby function
and I'll be able to add the "SLE migrated" one as you request.